flytekit/core/constants.py

```diff
@@ -9,3 +9,6 @@
 
 START_NODE_ID = "start-node"
 END_NODE_ID = "end-node"
+
+# To override the default container image or the default base image in ImageSpec, set this environment variable.
+FLYTE_INTERNAL_IMAGE = "FLYTE_INTERNAL_IMAGE"
```

Comment:
The convention with the reset of the code base is `_ENV_VAR`: 

```suggestion
FLYTE_INTERNAL_IMAGE_ENV_VAR = "FLYTE_INTERNAL_IMAGE"
```

https://github.com/flyteorg/flytekit/blob/b98e9416aee298435fc01d8d78f962bea60fd062/flytekit/configuration/file.py#L17-L18

Reply:
Good catch, thanks!